### PR TITLE
adding github action workflow

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
/cc @lance

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
/hold
also if we wanted to go this way, would need to add the `GITHUBTOKEN` as a secret

File: .github/workflows/cronjobs.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  schedule:
+    # Every M-F at 12:00am run this job
+    - cron:  "0 0 * * 1-5"
+    
+jobs:
+  check-image-version:
+    uses: securesign/actions/.github/workflows/check-image-version.yaml@main
+    strategy:
+      matrix:
+        branch: [main, redhat-v1.3]
```
Comment (lance) [inline .github/workflows/cronjobs.yaml:13]: I don't think we need the redhat-v1.3 branch here.

File: .github/workflows/cronjobs.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    with:
+      branch: ${{ matrix.branch }}
```
Comment (lance) [inline .github/workflows/cronjobs.yaml:15]: You can now specify the images you want to have managed. See: https://github.com/securesign/cosign/blob/72aac2145c2c68863c211747831674b40f7b46ca/.github/workflows/cronjobs.yaml#L16

**Review by lance — COMMENTED** [review]
@Gregory-Pereira sorry this one sat so long!

**Comment by lance** [comment]
OMG nevermind - this is for the gamma release. Closing.

**Pull request closed without merging**